Calculator.java

```diff
@@ -0,0 +1,42 @@
+/**
+ * make 4 simpliest math methods(+*-/) with 2 numbers from command line
+ * @author Babak Alexey
+ *
+ */
+public class Calculator {
+
+   /**
+   *check possibility of the "String -> Double" conversion 
+   */
+   public static boolean checkString(String string) {
+      try {
+         Double.parseDouble(string);
+      } catch (Exception e) {
+           return false;
+      }
+      return true;
+   }
+
+   /** 
+   *make 4 simpliest math methods(+*-/) with 2 numbers from command line
+   */
+   public static void main(String [] args) {
+      if(args.length != 2) {
+         System.out.println("Try to input 2 numbers");
+      } else if(!checkString(args[0]) || !checkString(args[1])) {
+         System.out.println("Try to input 2 NUMBERS");

```

Comment:
You can exit from main here - it'll make next else useless so it simplifies code and indentation